docs/release.rst

```diff
@@ -47,6 +47,9 @@ If you don't have nox, you should either use ``pipx run nox`` instead, or use
 - Update tags (optional; if you skip this, the GitHub release makes a
     non-annotated tag for you)
     - ``git tag -a vX.Y.Z -m 'vX.Y.Z release'``.
+    - To ensure the new tag is consistent with the version number in the sources, run this command:
```

Comment:
Where is the current tag number shown? `__version__` and `pybind11/_version.py` are already forced to be in sync by `nox -s tests_packaging` above, if that's all this is doing.

Reply:
The nox command only ensures that common.h and pybind11/_version.py are fully self-consistent.

Which is something I take for granted.

The git-diff-grep command would have shown a diff because it would have pulled `v2.11.0` from pybind11/_version.py, but I at that step in the instructions my working copy had the v2.11.1 changes, just not the version number update.

Reply:
I'm failing to understand why this would catch anything that `git status` or a plain `git diff` would not show (or, in my terminal, the current git condition is shown as a colored bar). It's not using or comparing the tag itself at all.

Could it be replaced with "make sure all changes are committed"?

Reply:
I simply forgot to update the version number, and everybody else overlooked it, too.

`git status` would not have shown anything.

`git diff` also would not have shown anything.

`git diff v2.11.0` would have shown that the code I had was not what was tagged as v2.11.0.

Reply:
Oooh, oooookay. That seems like an interesting way to so it. Can't you just print out the current version and make sure it's the same as the tag? `grep ^__version__ pybind11/_version.py | cut -d'"' -f2 | sed 's/^/v/'"` should show the current tag.

Reply:
And `grep ^__version__ pybind11/_version.py` would be enough for a human user to verify the code states the version they just tagged.